### ISSUE TITLE
[🐛 Bug]: Is it possible to run Selenium tests in headful mode using GitHub Actions?

### ISSUE THREAD

**Issue by AshokKumarSMC** [SeleniumHQ/selenium#14471]
### What happened?

I’m currently running Selenium test scripts using GitHub Actions, but the tests are executed in headless mode by default. I would like to know if it’s possible to run these tests in headful mode, where the browser is fully visible during the test execution.

**Questions:**

    Is it possible to configure GitHub Actions to run Selenium tests in headful mode?
    If yes, what are the steps or configurations required to achieve this?
    Are there any limitations or considerations when running Selenium tests in headful mode on GitHub Actions?

**Additional Information:**

    I am using the following setup:
    Java (version 17)
    Selenium WebDriver 
    Maven
    ChromeDriver

### How can we reproduce the issue?

```shell
My goal is to:

Visually observe the browser interactions during the test runs.
Ensure that elements are correctly rendered and interacted with as they would be in a normal browser session.

Any guidance or examples would be greatly appreciated. Thank you for your support!
```

### Relevant log output

```shell
NA
```

### Operating System

Windows 11

### Selenium version

 4.17.0

### What are the browser(s) and version(s) where you see this issue?

Chrome 128

### What are the browser driver(s) and version(s) where you see this issue?

Version 128.0.6613.119 (Official Build) (64-bit)

### Are you using Selenium Grid?

NO

**Comment by dtopuzov** [comment]
Yes, it is possible.
On `windows-latest` it works out of the box, on `ubuntu-latest` runners you can use `xvfb` like that:
https://github.com/dtopuzov/sikuli-actions/blob/main/.github/workflows/gradle.yml#L41-L44

**Comment by AshokKumarSMC** [comment]
> Yes, it is possible. On `windows-latest` it works out of the box, on `ubuntu-latest` runners you can use `xvfb` like that: https://github.com/dtopuzov/sikuli-actions/blob/main/.github/workflows/gradle.yml#L41-L44

I have tried using xvfb during the "Build with Maven" step, but it still isn't working for me. In my Selenium test script, I removed the headless argument from the ChromeOptions as shown below:

ChromeOptions options = new ChromeOptions();
// options.addArguments("--headless");  // I have removed this line
driver = new ChromeDriver(options);

However, I still can't visually observe the browser interactions during the tests.

Is there something else I need to configure or check? Any advice on how to ensure the browser is displayed and the tests run in headful mode?
